### PR TITLE
Editorial: refactor processing to use ordered map

### PR DESCRIPTION
**Pull request by marcoscaceres**
Aligns with how processing is done in Web Manifest, which is now expecting an ordered map to be returned.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/image-resource/pull/36.html" title="Last updated on Mar 9, 2021, 5:04 AM UTC (6fb0fb0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/image-resource/36/a9e599f...6fb0fb0.html" title="Last updated on Mar 9, 2021, 5:04 AM UTC (6fb0fb0)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -292,55 +292,59 @@ <h2>
...
-            <li>If |input|'s "type" property value is not a [=valid MIME type
-            string=], return failure.
+            <li>Let |mime| be the result of [=parse a mime type|parsing=]
+            |input|["type"].
+            </li>
+            <li>If |mime| is failure, return failure.
             </li>
-            <li>Set |image|'s [=image resource/type=] to the value of |input|'s
-            "type" property.
+            <li>Set |image|["type"] to the [=essence=] of |mine|.
```
Comment (aarongustafson) [inline index.html:334]: mine -> mime

**Review by aarongustafson — CHANGES_REQUESTED** [review]
LGTM with one change

**Comment by aarongustafson** [comment]
There are also several ReSpec errors that need resolution prior to merging.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:334]: Missed opportunity! 
Screen Shot 2021-03-08 at 12 05 35 pm

**Comment by marcoscaceres** [comment]
> There are also several ReSpec errors that need resolution prior to merging.

Will send a PR for those.

**Review by aarongustafson — APPROVED** [review]

**Pull request merged**